lib/content-services/src/lib/search/services/base-query-builder.service.ts

```diff
@@ -73,8 +73,14 @@ export abstract class BaseQueryBuilderService {
     /*  Stream that emits the initial value for some or all search filters */
     populateFilters = new BehaviorSubject<{ [key: string]: any }>({});
 
+    /*  Stream that emits every time queryFragments change */
+    queryFragmentsUpdate = new BehaviorSubject<{ [key: string]: any }>({});
```

Comment:
similar comment for any: 
https://github.com/Alfresco/alfresco-ng2-components/pull/11386/files#r2567686029